src/Razor/src/Microsoft.VisualStudio.RazorExtension/css-language-configuration.json

```diff
@@ -51,5 +57,14 @@
     "increaseIndentPattern": "(^.*\\{[^}]*$)",
     "decreaseIndentPattern": "^\\s*\\}"
   },
-  "wordPattern": "(#?-?\\d*\\.\\d\\w*%?)|(::?[\\w-]*(?=[^,{;]*[,{]))|(([@#.!])?[\\w-?]+%?|[@#!.])"
+  "wordPattern": "(#?-?\\d*\\.\\d\\w*%?)|(::?[\\w-]*(?=[^,{;]*[,{]))|(([@#.!])?[\\w-?]+%?|[@#!.])",
+  "onEnterRules": [
+    {
+      "beforeText": "^\\s*\\/\\*(?!\\/)([^\\*]|\\*(?!\\/))*$",
+      "afterText": "^\\s*\\*\\/$",
+      "action": {
+        "indent": "indentOutdent"
+      }
+    }
```

Comment:
Too bad we can't do trailing `,` in json.

Reply:
lol ya technically we shouldn't be allowed to do comments either but a lot of JSON parsing engines these days allow it (VSCode's + Newtonsoft) thankfully.